src/components/PendingNwc.tsx

```diff
@@ -81,6 +81,14 @@ export function PendingNwc() {
         }
     }
 
+    async function approveAll() {
+        // clone the list so it doesn't update in place
+        const toApprove = [...pendingRequests()!];
+        for (const item of toApprove) {
+            await payItem(item);
```

Comment:
we should handle if this fails, at the very least log it. Adding specific errors to the item would be ideal but that might be harder

Reply:
each "payitem" call handles its own errors